docs/.jekyll-cache/Jekyll/Cache/Jekyll--Cache/b7/9606fb3afea5bd1609ed40b622142f1c98125abcfe89a76a661b0e8e343910

```diff
@@ -0,0 +1 @@
js/Turing.jl/docs/_site", "collections_dir"=>"", "cache_dir"=>".jekyll-cache", "plugins_dir"=>"_plugins", "layouts_dir"=>"_layouts", "data_dir"=>"_data", "includes_dir"=>"_includes", "collections"=>{"posts"=>{"output"=>true, "permalink"=>"/:collection/:path"}, "docs"=>{"output"=>true, "permalink"=>"/:collection/:path"}, "tutorials"=>{"output"=>true, "permalink"=>"/:collection/:path"}, "team"=>{"output"=>true, "permalink"=>"/:collection/:path"}}, "safe"=>false, "include"=>[".htaccess"], "exclude"=>[".gems", "_site", "CHANGELOG.md", "CNAME", "LICENSE", "src", "Gemfile", "Gemfile.lock", "jekyll-build", "make.jl", "make-utils.jl", "README.md", "vendor", "test.sh", "*.org", ".sass-cache", ".jekyll-cache", "gemfiles", "node_modules", "vendor/bundle/", "vendor/cache/", "vendor/gems/", "vendor/ruby/"], "keep_files"=>[".git", ".svn"], "encoding"=>"utf-8", "markdown_ext"=>"markdown,mkdown,mkdn,mkd,md", "strict_front_matter"=>false, "show_drafts"=>nil, "limit_posts"=>0, "future"=>false, "unpublished"=>false, "whitelist"=>[], "plugins"=>["jekyll-paginate", "jekyll-sitemap", "jekyll-gist", "jekyll-feed", "jemoji", "jekyll-sitemap"], "markdown"=>"kramdown", "highlighter"=>"rouge", "lsi"=>false, "excerpt_separator"=>"\n\n", "incremental"=>false, "detach"=>false, "port"=>"4000", "host"=>"127.0.0.1", "baseurl"=>"/dev", "show_dir_listing"=>false, "permalink"=>"/:year/:title/", "paginate_path"=>"/page:num", "timezone"=>nil, "quiet"=>false, "verbose"=>false, "defaults"=>[{"scope"=>{"path"=>"_docs", "type"=>"docs"}, "values"=>{"layout"=>"page", "sidebar"=>true}}, {"scope"=>{"path"=>"_tutorials", "type"=>"tutorials"}, "values"=>{"layout"=>"page", "sidebar"=>true}}, {"scope"=>{"path"=>"", "type"=>"pages"}, "values"=>{"layout"=>"page", "sidebar"=>true}}, {"scope"=>{"path"=>"_team", "type"=>"team"}, "values"=>{"layout"=>"team", "sidebar"=>false}}, {"scope"=>{"path"=>"_posts", "type"=>"posts"}, "values"=>{"layout"=>"post", "sidebar"=>false}}], "liquid"=>{"error_mode"=>"warn", "strict_filters"=>false, "strict_variables"=>false}, "kramdown"=>{"auto_ids"=>true, "toc_levels"=>[1, 2, 3, 4, 5, 6], "entity_output"=>"as_char", "smart_quotes"=>"lsquo,rsquo,ldquo,rdquo", "input"=>"GFM", "hard_wrap"=>false, "guess_lang"=>true, "footnote_nr"=>1, "show_warnings"=>false}, "title"=>"Turing.jl", "email"=>"support@turing.jl", "author"=>"The Turing Team", "description"=>"Turing: A robust, efficient and modular library for general-purpose probabilistic programming.\n", "url"=>"http://localhost:4000", "repo"=>"https://github.com/TuringLang/Turing.jl", "github_user"=>"TuringLang", "github_repo"=>"Turing.jl", "twitter"=>"Turingjl", "linkedin"=>"Turingjl", "dockerhub"=>"Turingjl", "logo"=>"assets/img/favicon.ico", "logo_pixels"=>34, "color"=>"#d2e6f5", "navbarcolor"=>"#073c44", "tag_search_endpoint"=>"https://ask.cyberinfrastructure.org/search?q=", "tag_color"=>"danger", "accentColor"=>"red", "themeColor"=>"red", "fixedNav"=>"true", "livereload_port"=>35729, "serving"=>true, "watch"=>true}:ET
```

Comment:
@denainjs Perhaps consider removing these immediate files from Jekyll?

Reply:
Not sure why `.jekyll-cache` was checked in. It should be removed from the PR and we should probably add `.jekyll-cache` to the `.gitignore` file.